app/shared/detectLanguage.ts

```diff
@@ -1,5 +1,6 @@
 import franc from 'franc';
-import get from 'shared/languagesList';
+import { language as get } from 'shared/languagesList';
```

Comment:
Get is really a very poor name here!  I get it you are just reassigning the import, but this one is really a sucky name. :(

Reply:
Fixed